packages/gatsby-theme-newrelic/package.json

```diff
@@ -11,6 +11,14 @@
     "clean": "gatsby clean",
     "develop": "gatsby develop"
   },
+  "homepage": "https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic",
+  "repository": "https://github.com/newrelic/gatsby-theme-newrelic",
+  "keywords": [
+    "gatsby",
+    "gatsby-theme",
+    "gatsby-plugin",
+    "react"
+  ],
```

Comment:
should newrelic be a keyword?

Reply:
I was just thinking the same thing this morning. I'll add it.